### PR TITLE
feat(onboarding): remove welcome page experiment and use new experience

### PR DESCRIPTION
**Pull request by scefali**
This PR copies the welcome page component from the targeted onboarding flow into the default onboarding flow and removes the `TargetedOnboardingWelcomePageExperimentV2` experiment. There are some minor differences to handle the different prop types but everything else is the same.

Since we only have one welcome background now, I was able to delete the one from the targeted onboarding folder and just copy its contents to the shared one.

Here's what it looks like now:
Screen Shot 2022-04-14 at 9 35 46 AM

### CODE REVIEW

File: static/app/views/onboarding/types.tsx [hunk 1]
```diff
@@ -10,9 +10,9 @@ export type StepProps = {
   onComplete: (data: StepData) => void;
   onUpdate: (data: StepData) => void;
   orgId: string;
+  organization: Organization;
   platform: PlatformKey | null;
   project: Project | null;
-  organization?: Organization;
```
Comment (scefali) [inline static/app/views/onboarding/types.tsx:15]: I'm not sure why `organization` was marked as optional as it makes typing more difficult and the parent component has organization as required. So I just made it required.

**Review by PixelDust22 — APPROVED** [review]
LGTM

File: static/app/views/onboarding/targetedOnboarding/components/welcomeBackground.tsx [hunk 2]
```diff
@@ -1,277 +0,0 @@
...
-    display: none;
-  }
-`;
-Container.defaultProps = {
-  variants: {
-    animate: {},
-    exit: {},
-  },
-  transition: testableTransition({staggerChildren: 0.2}),
-};
```
Comment (evanpurkhiser) [inline static/app/views/onboarding/targetedOnboarding/components/welcomeBackground.tsx:277]: Now that you've deleted my nicely animated old background.

Can you please make the new background have some nice animations :)

File: tests/js/spec/views/onboarding/welcome.spec.jsx [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
+import {initializeOrg} from 'sentry-test/initializeOrg';
 
 import ConfigStore from 'sentry/stores/configStore';
 import OnboardingWelcome from 'sentry/views/onboarding/welcome';
 
 describe('OnboardingWelcome', function () {
+  const {organization} = initializeOrg({});
   it('renders', function () {
-    const name = 'Rick Snachez';
+    const name = 'Rick Sanchez';
```
Comment (evanpurkhiser) [inline tests/js/spec/views/onboarding/welcome.spec.jsx:10]: good fix

**Pull request merged**